tests/conftest.py

```diff
@@ -14,6 +14,16 @@
 from any_agent.tracing.agent_trace import AgentTrace
 from tests.integration.helpers import wait_for_server_async
 
+PATCH_PER_FRAMEWORK = {
```

Comment:
Should this constant be moved to a conftest.py inside of the unit/tools section?

Also, it may be useful to give this a more descriptive constant name, PATCH_PER_FRAMEWORK isn't very descriptive.

Reply:
Ok